examples/release_highlights/plot_release_highlights_1_2_0.py

```diff
@@ -82,6 +82,32 @@
 )
 X.head()
 
+# %%
+# New and enhanced displays
+# -------------------------
+# The :class:`~metrics.PredictionErrorDisplay` provides a one way to analyze
+# regression models in a qualitative manner.
+import matplotlib.pyplot as plt
+from sklearn.metrics import PredictionErrorDisplay
+
+X, y = load_diabetes(return_X_y=True, as_frame=True)
```

Comment:
I think it's fine to reuse the `hist_no_interact` model fitted in the "Interaction constraints in Histogram-based Gradient Boosting Trees" section but then I would move the section on the display classes right after it to avoid having to re-load `X` and `y`.